schemas/questionnaire_v1.json

```diff
@@ -255,18 +239,27 @@
                 }
               },
               "additionalProperties": false,
-              "required": [
-                "id",
-                "title",
-                "blocks"
-              ]
+              "required": ["id", "title", "blocks"]
             }
+          },
+          "enabled": {
```

Comment:
I guess it might be overkill but `enabled` here is dependant on the `hub` being `enabled`.

Reply:
Discussed with @ajmaddaford that they are independent and they can be used in cases without the notion of a hub.